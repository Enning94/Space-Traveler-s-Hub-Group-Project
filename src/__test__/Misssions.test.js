import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Missions from '../Components/Pages/MissionsPage';
import store from '../Redux/configureStore';

describe('Renders correctly', () => {
  it('Renders page correctly', () => {
    const { container } = render(<Provider store={store}><Missions /></Provider>);
    expect(container).toMatchSnapshot();
  });
});
