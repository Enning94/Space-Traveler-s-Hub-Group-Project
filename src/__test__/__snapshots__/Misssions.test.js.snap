// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly Renders page correctly 1`] = `
<div>
  <table
    class="wrapper mt-10"
  >
    <thead>
      <tr>
        <th
          class="font-important"
        >
          Mission
        </th>
        <th
          class="description font-important"
        >
          Description
        </th>
        <th
          class="font-important"
        >
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
